cppConsole/.vscode/tasks.json

```diff
@@ -123,6 +123,7 @@
             "args": [
                 "docker",
                 "build",
+                "--pull",
```

Comment:
hmm, why are we not using the commons on `cppConsole`?

Reply:
Those are tasks for building the SDK container, which weren't on the commons, I don't know why

Reply:
I will take a look if they can go to common

Reply:
I added a commit with this now